tests/testthat/test-verb-filter.R

```diff
@@ -29,6 +29,12 @@ test_that("each argument gets implicit parens", {
   expect_equal_tbl(mf1, mf2)
 })
 
+test_that("only add step if necessary", {
+  lf <- lazy_frame(x = 1:3, y = 1:3)
+  expect_equal(lf %>% filter(), lf)
+  expect_equal(lf %>% filter(!!!list()), lf)
```

Comment:
Why can't this be `lf %>% filter()`?

Reply:
Hmm, there was a case where `some_verb()` (I don't know which verb it was) and `some_verb(!!!list())` was different. But I don't see where this should have been the case so I remove these tests.